### ISSUE TITLE
tutorial example will not run on either Windows 7 or Ubuntu under Anaconda Python 3

### ISSUE THREAD

**Issue by empirical-bayesian** [gpoulter/python-ngram#9]
```
Python 3.4.3 |Anaconda custom (64-bit)| (default, Mar  6 2015, 12:06:10) [MSC v.1600 64 bit (AMD64)]
Type "copyright", "credits" or "license" for more information.

IPython 5.1.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import ngram

In [2]: ngram.Ngram.compare('Ham', 'Spam', N=1)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-c0c27ad8f009> in <module>()
----> 1 ngram.Ngram.compare('Ham', 'Spam', N=1)

AttributeError: 'module' object has no attribute 'Ngram'

In [3]:
```

**Comment by gpoulter** [comment]
Hi Jan! Thank you for the report.  I'm going to try reproduce it and update NGram - it's been a long time and it's possible bit-rot has set in wrt newer Python versions!

**Comment by gpoulter** [comment]
Seems to work fine under Python 3.6, trying Anaconda Python 3.4.3 (Pyton 3.6.0) on Windows 10 64-bit

$ pip install -m ngram
$ python
Python 3.6.0 |Anaconda 4.3.0 (64-bit)| (default, Dec 23 2016, 11:57:41) [MSC v.1900 64 bit (AMD64)] on win32
>>> import ngram
>>> ngram.NGram.compare('Ham', 'Spam', N=1)
0.4

Similarly in iPython:

C:\Users\graha_000>ipython
Python 3.6.0 |Anaconda 4.3.0 (64-bit)| (default, Dec 23 2016, 11:57:41) [MSC v.1900 64 bit (AMD64)]
Type "copyright", "credits" or "license" for more information.
IPython 5.1.0 -- An enhanced Interactive Python.
In [1]: import ngram
In [2]: ngram.NGram.compare('Ham', 'Spam', N=1)
Out[2]: 0.4


Perhaps you could elaborate on how you installed NGram?

**Comment by empirical-bayesian** [comment]
I don't think the problem is with _ngram_.  I think the problem is something broken in my Anaconda environment.  Oddly, it is the same breakage on both Ubuntu and Windows 7. I can import _ngram_ fine, but then, whether I invoke it from _ipython_ or directly from the shell, it cannot find the name `Ngram` in the package. I ran into something like this once before, and it was fixed, but, then, I don't know exactly how.  I have tried to update everything using the standard conda update mechanisms. 

Odd.
